### PR TITLE
Add text for the iOS Tutorial

### PR DESCRIPTION
**Pull request by cbush**
- Backend setup needed.
- Screenshots needed.
- Text mostly good.
- Based on https://docs.mongodb.com/stitch/tutorials/guides/todo-guide-1/ and https://docs.realm.io/sync/getting-started-1/step-1-my-first-realm-app

## Staged

https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/ios-tutorial/tutorial/ios-swift.html

## JIRA

https://jira.mongodb.org/browse/DOCSP-10293

### CODE REVIEW

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 1]
```diff
@@ -0,0 +1,901 @@
+---
+title: Create a New iOS Project
+ref: create-a-new-ios-project
+content: |
+  Open Xcode and create a new iOS Project. Use the "Single View App" template. Enter your project details and save.
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:5]: I'd recommend encouraging them not to select unit tests.

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 2]
```diff
@@ -0,0 +1,901 @@
...
+          use_frameworks!
+
+          # Pods for TaskTracker
+          pod 'RealmSwift', '=10.0.0-alpha.4'
+
+          target 'TaskTrackerTests' do
+            inherit! :search_paths
+            # Pods for testing
+            pod 'RealmSwift', '=10.0.0-alpha.4'
+          end
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:52]:
```suggestion
```

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 3]
```diff
@@ -0,0 +1,901 @@
...
+        end
+
+  Save your changes, then run the following command from the terminal window to download the required packages and create an Xcode workspace that is configured:
+
+  .. code-block:: text
+
+     pod install --repo-update
+
+  Open the Xcode workspace file that CocoaPods just created to continue working on your project. For example, you can run ``open TaskTracker.xcworkspace`` from the command line.
+  Build and run the project again to verify that everything is configured correctly.
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:63]: I **had** to run it here in order for everything to link up so it wouldn't error when I tried to import RealmSwift. Maybe make running it again a next step

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 4]
```diff
@@ -0,0 +1,901 @@
...
+         static let REALM_APP_ID = "Realm app ID here"
+       
+         // The base URL points to the server hosting MongoDB Realm.
+         static let REALM_BASE_URL = "https://realm-dev.mongodb.com"
+     }
+
+     let app = RealmApp(Constants.REALM_APP_ID, configuration: AppConfiguration(baseURL: Constants.REALM_BASE_URL,
+        transport: nil,
+        localAppName: nil,
+        localAppVersion: nil))
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:117]: When I formatted the code here with the build in XCode formatter it lined all of the function args up on the right.

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 5]
```diff
@@ -0,0 +1,901 @@
...
+
+     import RealmSwift
+
+     struct Constants {
+         // Set this to your Realm App ID found in the Realm UI.
+         static let REALM_APP_ID = "Realm app ID here"
+       
+         // The base URL points to the server hosting MongoDB Realm.
+         static let REALM_BASE_URL = "https://realm-dev.mongodb.com"
+     }
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:112]: do we want this here or in a constants file as we discussed with the code?

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 6]
```diff
@@ -0,0 +1,901 @@
...
+  Next, add a ``viewDidLoad`` method to the view controller.
+  This is where we will set up the login UI
+  programmatically. This just arranges and configures the
+  inputs for username and password, sign in and sign up
+  buttons, an area for a status or error message and a
+  loading indicator.
+
+  .. note::
+  
+     Excited for SwiftUI? So are we! A tutorial using
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:221]: ❤️

File: source/includes/steps-tutorial-ios-swift.yaml [hunk 7]
```diff
@@ -0,0 +1,901 @@
...
+         }))
+         alertController.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
+         self.present(alertController, animated: true, completion: nil)
+     }
+
+  This simply calls the ``logOut`` method on the app we
+  created earlier. In the completion handler, we dispatch to
+  the main thread so that we can navigate back to the welcome screen.
+---
+title: Implement the UITableViewDelegate and UITableViewDataSource protocols
```
Comment (terakilobyte) [inline source/includes/steps-tutorial-ios-swift.yaml:629]:
```suggestion
title: Implement the UITableViewDelegate and UITableViewDataSource Protocols
```

File: source/includes/steps-tutorial-ios-swift.yaml (hunk 5 above)
Comment (cbush) [reply source/includes/steps-tutorial-ios-swift.yaml:112]: No, I updated the code to remove that file. Creating a constants file is an unnecessary step. Arguably, you have to do it in Obj-C, so maybe. However, it makes sense for Swift to just put it in the entry point file. Thoughts?

Comment (terakilobyte) [reply source/includes/steps-tutorial-ios-swift.yaml:112]: I'm not highly opinionated either way, but let's make sure to update the repo code to match the tutorial.

File: source/includes/steps-tutorial-ios-swift.yaml (hunk 4 above)
Comment (cbush) [reply source/includes/steps-tutorial-ios-swift.yaml:117]: Not sure. Definitely off by one space due to rst's 3 space default.

File: source/includes/steps-tutorial-ios-swift.yaml (hunk 5 above)
Comment (cbush) [reply source/includes/steps-tutorial-ios-swift.yaml:112]: PR for that: https://github.com/mongodb-university/realm-tutorial/pull/2

File: source/tutorial/ios-swift.txt [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+This tutorial should take around 30 minutes.
+
+.. note::
+   
+   Want to get started right away with the complete source
+   code? Check out our :github:`repo
+   <mongodb-university/realm-tutorial>` with
+   ready-to-compile source code, then follow the instructions
+   in README.md to get started. Don't forget to update the
+   AppDelegate.swift file with your {+app+} ID, which you
```
[the remark this answers is not in the document]
Comment (cbush) [reply source/tutorial/ios-swift.txt:34]: Decided to leave it in AppDelegate for now.

**Pull request merged**